Domains/Smartmetering/README.md

```diff
@@ -29,10 +29,10 @@ Smart metering features available within on the open smart grid platform so far:
 #### SmartMeteringAdHoc
 - **[SynchronizeTime](./smartmeteringwebservices/SynchronizeTime.md)** is a request to synchronize the date and time on a device. The date and time are retrieved from the server and sent to the device.
 - **[GetSynchronizeTimeResponse](./smartmeteringwebservices/GetSynchronizeTimeResponse.md)** is a request which returns the response from the [SynchronizeTime](./smartmeteringwebservices/SynchronizeTime.md) request.
-- **[RetrieveConfigurationObjects](./smartmeteringwebservices/RetrieveConfigurationObjects.md)** is a request to obtain the entire tree and list of objects from an E-meter. 
-- **[GetRetrieveConfigurationObjectsResponse](./smartmeteringwebservices/GetRetrieveConfigurationObjectsResponse.md)** is a request which returns the response from the [RetrieveConfigurationObjects](./smartmeteringwebservices/RetrieveConfigurationObjects.md) request.
-- **[GetSpecificConfigurationObject](./smartmeteringwebservices/GetSpecificConfigurationObject.md)** is a request to obtain a specific object from an E-meter. 
-- **[GetSpecificConfigurationObjectResponse](./smartmeteringwebservices/GetSpecificConfigurationObjectResponse.md)** is a request which returns the response from the [GetSpecificConfigurationObject](./smartmeteringwebservices/GetSpecificConfigurationObject.md) request.
+- **[RetrieveAllAttributeValues](./smartmeteringwebservices/RetrieveAllAttributeValues.md)** is a request to obtain the entire tree and list of objects from an E-meter. 
+- **[GetRetrieveAllAttributeValuesResponse](./smartmeteringwebservices/GetRetrieveAllAttributeValuesResponse.md)** is a request which returns the response from the [RetrieveAllAttributeValues](./smartmeteringwebservices/RetrieveAllAttributeValues.md) request.
+- **[GetSpecificAttributeValue](./smartmeteringwebservices/GetSpecificAttributeValue.md)** is a request to obtain a specific object from an E-meter. 
```

Comment:
If we change the method name, we should also change the description, hence replace 'list of objects' accordingly. The same applies for line 34